RecoEgamma/EgammaElectronProducers/python/ecalDrivenElectronSeeds_cfi.py

```diff
@@ -34,11 +34,18 @@
 from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
     ecalDrivenElectronSeeds,
-    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal')
+    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCal'),
+    SeedConfiguration = dict( allowHGCal = cms.bool(True) )
 )
+
+
+# create ecal driven seeds for electron using HGCal Multiclusters
+ecalDrivenElectronSeedsFromMultiCl = ecalDrivenElectronSeeds.clone()
+
+from Configuration.Eras.Modifier_phase2_hgcal_cff import phase2_hgcal
 phase2_hgcal.toModify(
-    ecalDrivenElectronSeeds.SeedConfiguration,
-    allowHGCal = cms.bool(True)
+    ecalDrivenElectronSeedsFromMultiCl,
+    endcapSuperClusters = cms.InputTag('particleFlowSuperClusterHGCalFromMultiCl'),
+    SeedConfiguration = dict( allowHGCal = cms.bool(True) )
```

Comment:
the SeedConfiguration line is not necessary: ecalDrivenElectronSeedsFromMultiCl should now be the same as the phase2_hgcal version of ecalDrivenElectronSeeds